### PR TITLE
github: add job to check privnet image build

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Related to
https://github.com/nspcc-dev/neo-go/pull/2408#pullrequestreview-921289314.

### CODE REVIEW

**Comment by AnnaShaleva** [comment]
@roman-khimov, currently we do not support building privnet image on Windows. Do we need to extend our Makefile to support it?

**Review by fyrchik — APPROVED** [review]

**Comment by AnnaShaleva** [comment]
I still need to check it on my Windows machine, will do this in the evening.

File: Makefile [hunk 1]
```diff
@@ -131,3 +132,32 @@ env_restart:
 env_clean: env_down
 	@echo "=> Cleanup environment"
 	@docker volume rm docker_volume_chain
+
+# --- Windows Server Core environment ---
+env_image-wsc:
```
Comment (roman-khimov) [inline Makefile:137]: Can we detect Windows and handle this in the `env_image` target?

Comment (AnnaShaleva) [reply Makefile]: Fixed!

File: docs/consensus.md [hunk 2]
```diff
@@ -147,7 +147,8 @@ four-node setup.
 - `docker-compose`
 - `go` compiler
 
-#### Instructions
+#### Instructions (Ubuntu / WindowsServerCore)
```
Comment (roman-khimov) [inline docs/consensus.md:150]: Does it worth changing?

File: docs/consensus.md [hunk 3]
```diff
@@ -143,11 +143,12 @@ Node wallets are located in the `.docker/wallets` directory where
 four-node setup.
 
 #### Prerequisites
-- `docker`
+- `docker` of version >= 20.10.0
 - `docker-compose`
 - `go` compiler
 
 #### Instructions
+
```
Comment (roman-khimov) [inline docs/consensus.md:151]: :eyes:

Comment (AnnaShaleva) [reply docs/consensus.md]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**